js/articleView.js

```diff
@@ -1,3 +1,14 @@
+// fade in the page by removing the class
+$(function() {
+    $('body').removeClass('fade-out');
+});
+
+// for smaller screens, show menu option when clicking the hambuger icon
+function showMenuOptions() {
```

Comment:
These are more typically handled by the CSS and responsive design for differing viewports.